assets/src/edit-story/components/panels/design/preset/colorPreset/colorPresetActions.js

```diff
@@ -183,13 +193,16 @@ function ColorPresetActions({ color, pushUpdate }) {
     <ActionsWrapper>
       <HeaderRow>
         <DropDownWrapper>
-          <AdvancedDropDown
+          <StyledDropDown
```

Comment:
The style of the button to open the dropdown looks a bit different from Figma and it seems like styling it is not really straightforward since it's using this custom style only for the color picker.

 E.g. the chevron has to be right next to the text here, unlike in case of other dropdowns:
![Screenshot 2021-03-08 at 12 14 15](https://user-images.githubusercontent.com/3294597/110307686-7e764400-7fff-11eb-92bb-5104eb0ef77a.png)

Currently, the text area is fixed and the chevron on the right end.

Maybe we should still use the `Menu` directly here? Or perhaps we should create a separate Simplified Dropdown component.

(We could leave it for a follow-up PR)

Reply:
Seems like a good case for just using the `Menu` directly (and not in a popup, but inline). I'll look at implementing directly.

Reply:
Another option is to make the menu style override-able. We had this happen once in the dashboard as well. 

My initial thought in how I implemented the dropDown was that there should be uniformity. So if we could get design in on the 'variations' of the dropDown and then make 4 variants or what have you, that would be great. I want to try and tread this carefully so we don't get into a similar spot as before where we had all these slightly different dropDowns.  

In the essence  of time, i understand if this needs to be a separate ticket to clean up.

Reply:
For now, I just fixed the inline alignment in ea7fb065b. I don't see a great need for a generalized solution nor a follow-up?